backend/server.ts

```diff
@@ -0,0 +1,25 @@
+import cors from 'cors';
+import express from 'express';
+import todoRoutes from './routes/todoRoutes';
+import userRoutes from './routes/userRoutes';
+
+import config from 'dotenv';
+
+config.config({path: __dirname + '/.env'});
+
+const app = express();
+const port = 3001;
+
+app.use(express.json());
```

Comment:
Funguju ti v pohode PUT a POST akcie? Mozno by som este doporucil mrknut na metodu urlencoded() https://stackoverflow.com/questions/23259168/what-are-express-json-and-express-urlencoded

potom realne v kode len tu pridat riadok `app.use(express.urlencoded({ extended: true }));`

Reply:
`express.json()` staci, kdyz vracis vzdy objekt, kdyby jsi chtel vracet stringy/array, tak potrebujes `express.urlencoded({ extended: true })`, nam staci `.json()`